bugbug/model.py

```diff
@@ -285,6 +285,24 @@ def print_feature_importances(
                 end="\n\n",
             )
 
+    def save_feature_importances(self, important_features, feature_names):
+        # Returns a JSON-encodable dictionary that can be saved in the metrics
+        # report
+        top_feature_names = [
+            feature_names[int(index)]
+            for importance, index, is_pos in important_features["average"]
+        ]
+
+        feature_report = {}
+
+        for i, feature_name in enumerate(top_feature_names):
+            for class_name, imp_values in important_features["classes"].items():
+                feature_report.setdefault(feature_name, {})[class_name] = float(
+                    imp_values[1][i]
+                )
+
+        return feature_report
```

Comment:
We could save (and, in the next PR, check) the "average" features too in the report.